lib/shipcloud/order_return.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Shipcloud
+  class OrderReturn < Base
+    include Shipcloud::Operations::All
+    include Shipcloud::Operations::Find
```

Comment:
At the moment, we can't control which Operations are offered by Resource as a nested resource; only which operations it offers when used directly. I think the Resource should define that but it might also be a case of yagni

Reply:
good point! Although I'd say the parent resource (here the `Order`) should define the child routes, it would be best if the nested operations can be defined distinctly. I implemented a proposal in [7c2d48d](https://github.com/shipcloud/shipcloud-ruby/pull/122/commits/7c2d48d3f76345a1fea7230b1d5c7c927c9e27c8). In the future we could amend the nested operations class for general actions (e.g. update/destroy) once, but choose which operations are defined on the nested resources with the `#add` method.